### DISCUSSION TITLE
Ability to ignore default project filter

### DISCUSSION THREAD

**Discussion by ccmattr** [ankitpokhrel/jira-cli#261] [Ideas] [answered]
At the moment in our small company we have sprints spanning multiple projects, with a project dedicated to hosting the sprint board. When I do `jira sprint list` I get the interactive menu that shows me all the issues in the sprint, which is great. but when I try adding options like `--plain --table --current` it then adds in the default project filter from the config file which means I get no issues returned as none of the issues in the sprint are in the default project.

**Comment by ankitpokhrel** [comment]
@ccmattr could you please provide steps to reproduce this behavior in Jira?

**Comment by ankitpokhrel** [comment, marked as answer]
@ccmattr This feature is available in the latest beta release [v1.0.0-beta.2](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.0.0-beta.2). Details https://github.com/ankitpokhrel/jira-cli/pull/395.

For sprints, you can do
```
$ jira sprint list --current -q"project IS NOT EMPTY"
```
